index.html

```diff
@@ -3,14 +3,85 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta content="width=device-width, initial-scale=1">
+
+
   <title>Great Idea - Responsive I</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
 </head>
+<div class="container">
+  <div class="nav">
+<nav class="navhead">
+    <a href="services.html">Services</a>
+    <a href=#servprodvision>Product</a>
```

Comment:
please note that the way these Atags are formatted, they will not link to anything. You are missing a few elements.